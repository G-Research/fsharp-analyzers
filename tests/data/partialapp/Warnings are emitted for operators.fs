module M

let partapp3 = (+) 4 55 // should not warn
let partapp4 = (+) 4 // should warn
