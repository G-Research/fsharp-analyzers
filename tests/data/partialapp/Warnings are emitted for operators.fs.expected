GRA-PARTAPP-001 | Warning | (4,16 - 4,17) | partial application should not be used: op_Addition at (4,16--4,17)
